contrib/generate_precompile.jl

```diff
@@ -465,6 +465,7 @@ generate_precompile_statements() = try # Make sure `ansi_enablecursor` is printe
     print("Total ─────── "); Base.time_print(stdout, tot_time); println()
 finally
     fancyprint && print(ansi_enablecursor)
+    GC.gc(true) # reduce memory footprint
```

Comment:
Perhaps replace this with `GC.gc(true); GC.gc(false)`?

`GC.gc(true)` just triggers a full sweep, which means that old garbage will be collected in the next GC cycle.

We might want to insert a `GC.gc(false)` to expedite this subsequent collection of garbage.

Reply:
Don't we do a recollect with the manual GC?